NetKVM/Common/ParaNdis-Util.h

```diff
@@ -10,90 +11,88 @@ extern "C" {
 
-static __inline
-BOOLEAN IsPowerOfTwo(ULONG n)
+static __inline BOOLEAN IsPowerOfTwo(ULONG n)
 {
     return ((n != 0) && ((n & (~n + 1)) == n));
 }
 
 class CNdisAllocatableBase
 {
-protected:
-    CNdisAllocatableBase() {};
-    ~CNdisAllocatableBase() {};
+  protected:
+    CNdisAllocatableBase(){};
+    ~CNdisAllocatableBase(){};
     /* Objects's array can't be freed by NdisFreeMemoryWithTagPriority, as C++ array constructor uses the
-    * first several bytes for array length. Array  destructor can't get additional argument, so passing
-    * NDIS_HANDLE to the array destructor is impossible. Therefore, the array constructors and destructor
-    * are declared private and object arrays has be created and destroyed in two steps - construction by
-    * allocating array memory with NdisAllocateMemoryWithTagPriority and then in-place constructing the
-    * objects and destructing in the reverse order. */
-    void* operator new[](size_t Size, NDIS_HANDLE MiniportHandle) throw() = delete;
-    void* operator new[](size_t Size) throw() = delete;
+     * first several bytes for array length. Array  destructor can't get additional argument, so passing
+     * NDIS_HANDLE to the array destructor is impossible. Therefore, the array constructors and destructor
+     * are declared private and object arrays has be created and destroyed in two steps - construction by
+     * allocating array memory with NdisAllocateMemoryWithTagPriority and then in-place constructing the
+     * objects and destructing in the reverse order. */
+    void *operator new[](size_t Size, NDIS_HANDLE MiniportHandle) throw() = delete;
+    void *operator new[](size_t Size) throw() = delete;
     void operator delete[](void *) = delete;
 
-private:
+  private:
     /* The delete operator can't be disabled like array constructors and destructors, as default destructor
-    * and default constructor depend on the delete operator availability */
-    void operator delete(void *) {}
+     * and default constructor depend on the delete operator availability */
+    void operator delete(void *)
+    {
+    }
 };
 
 class CPlacementAllocatable
 {
-public:
-    void* operator new(size_t size, void *ptr) throw()
+  public:
+    void *operator new(size_t size, void *ptr) throw()
     {
         UNREFERENCED_PARAMETER(size);
         return ptr;
     }
 };
 
-template <typename T, ULONG Tag>
-class CNdisAllocatable : private CNdisAllocatableBase
+template <typename T, ULONG Tag> class CNdisAllocatable : private CNdisAllocatableBase
 {
-public:
-    void* operator new(size_t Size, NDIS_HANDLE MiniportHandle) throw()
-        { return NdisAllocateMemoryWithTagPriority(MiniportHandle, (UINT) Size, Tag, NormalPoolPriority); }
+  public:
+    void *operator new(size_t Size, NDIS_HANDLE MiniportHandle) throw()
+    {
+        return NdisAllocateMemoryWithTagPriority(MiniportHandle, (UINT)Size, Tag, NormalPoolPriority);
```

Comment:
@YanVugenfirer As we see sometimes `ContinuationIndentWidth` still does not format function to multiline call. 
I think in the next iteration we should check how it will look with:
```
AlignAfterOpenBracket: AlwaysBreak
ContinuationIndentWidth: 4
AllowAllArgumentsOnNextLine: false
AllowAllParametersOfDeclarationOnNextLine: false
```